### ISSUE TITLE
vertex_normal

### ISSUE THREAD

**Issue by amy-tipple-enview** [janba/GEL#27]
Hi folks, 
Thanks for the great work.  
I came across an error while installing `PyGEL3D` after the latest commit `94f6596843720b14adce44e7291b649b3a488ce3`

Ubuntu 18.04
`AttributeError: /usr/local/lib/python3.6/dist-packages/PyGEL3D/libPyGEL.so: undefined symbol: vertex_normal`

I read somewhere it might be the order in which libraries are being installed, but I didn't have any luck there.

Any suggestions?
Thanks

**Comment by janba** [comment]
Hi Amy,

Thanks for reporting this. I have changed the header files for the bridge between the C++ based GEL library and PyGEL to make them parsable by a C compiler, but I had forgotten to change a few functions (e.g. vertex_normal) to take only C type arguments … and then Python gets confused when you import.

It should be fixed now, but otherwise let me know.

Andreas

On 23 Jan 2020, at 16.55, amy-tipple-enview <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi folks,
Thanks for the great work.
I came across an error while installing PyGEL3D after the latest commit 94f6596843720b14adce44e7291b649b3a488ce3

Ubuntu 18.04
AttributeError: /usr/local/lib/python3.6/dist-packages/PyGEL3D/libPyGEL.so: undefined symbol: vertex_normal

I read somewhere it might be the order in which libraries are being installed, but I didn't have any luck there.

Any suggestions?
Thanks
